shared/types/graphql.ts

```diff
@@ -385,15 +384,15 @@ export type ParsedOOV1GraphEntity = {
   time: string;
   requester: Address;
   currency: Address;
-  reward: BigNumber;
-  finalFee: BigNumber;
+  reward: bigint;
+  finalFee: bigint;
```

Comment:
the biggest pain is that in the version wagmi stopped using ethers and bignumbers, switched to viem and bigint, so everything needs to be converted